src/storage-blob-preview/azext_storage_blob_preview/operations/blob.py

```diff
@@ -429,13 +430,20 @@ def copy_blob(client, source_url, metadata=None, **kwargs):
     return client.start_copy_from_url(source_url=source_url, metadata=metadata, incremental_copy=False, **kwargs)
 
 
-def download_blob(client, file_path, open_mode='wb', progress_callback=None, socket_timeout=None, **kwargs):
+def download_blob(client, file_path=None, open_mode='wb', progress_callback=None, socket_timeout=None, **kwargs):
     if progress_callback:
         kwargs['raw_response_hook'] = progress_callback
+    if not file_path:
+        kwargs['max_concurrency'] = 1
     download_stream = client.download_blob(**kwargs)
-    with open(file_path, open_mode) as stream:
-        blob = download_stream.readinto(stream)
-    return blob
+    if file_path:
+        with open(file_path, open_mode) as stream:
+            blob = download_stream.readinto(stream)
+        return blob
+    else:
+        with os.fdopen(sys.stdout.fileno(), open_mode) as stream:
+            download_stream.readinto(stream)
```

Comment:
`sys.stdout` was somehow overwritten from `colorama.ansitowin32.StreamWrapper` to `tempfile._TemporaryFileWrapper` by test framwork. While the latter one doesn't have `fileno()` operation, so I can't write successful tests for this scenario.